src/stream/turn_server.rs

```diff
@@ -15,6 +15,9 @@ use turn::Error;
 
 use log::*;
 
+pub const WEBRTCSINK_PLUGIN_NAME: &str = "webrtcsink";
+pub const WEBRTCSINK_VERSION: &str = "0.1.0";
```

Comment:
this webrtc specific stuff should be in a webrtc file, if turn server is webrtc only, for organization, we should move the necessary files in a stream/webrtc folder.